### DISCUSSION TITLE
'open_basedir restriction in effect' error

### DISCUSSION THREAD

**Discussion by jenhendrix** [pods-framework/pods#6662] [Q&A]
Just moved a client website on to the Kinsta cloud platform.
https://www.michiganmasons.org

We are receiving PHP warning messages from Pods plugin in regards to partial views that we are using in a custom theme. Partials are located under themes/custom/partials/

FastCGI sent in stderr: "PHP message: PHP Warning: file_exists(): open_basedir restriction in effect. File(/partials/cta-section.php) is not within the allowed path(s): (***ALLOWED PATHS HERE***) in /www/michiganmasonsorg_612/public/wp-content/plugins/pods/includes/general.php on line 1131" while reading response header from upstream, upstream: "fastcgi://unix:/var/run/php7.4-fpm-michiganmasonsorg.sock:", host: "michiganmasons.org"

It looks like that portion of code is checking if a file exists for $tags[view]

The partials path is not fully written out in the error log and I am wondering if that is why it cannot be located? Is there a way for me to update this path?

**Comment by JoryHogeveen** [comment]
Hi @jenhendrix 

Where is this `partials` directory located? Where are you using it?
It seems this is indeed an incorrect directory in your server.
